src/Makefile.in

```diff
@@ -42,7 +42,7 @@ top_srcdir = @top_srcdir@
 srcdir = @srcdir@
 prefix = @prefix@
 
-JTR_GIT_VERSION = \"$(shell git describe --tags --dirty=+ 2>/dev/null)\"
+JTR_GIT_VERSION = \"$(shell git describe --tags --dirty=+ --always 2>/dev/null)\"
```

Comment:
Where did you see a difference between these 2 commands? I don't.
Can you provide an example where the `--always` helps?
Is it for commits that are older than the first tag?

Is this output
```
((HEAD detached at 18553ff))src $ git describe --tags --dirty=+ --always
18553ff
```
instead of 
```
(bleeding-jumbo)src $ git describe --tags --dirty=+
1.8.0.6-jumbo-1-2261-g6421abe
```
really helpful?
If you checkout such an old version, you don't even have to old `git describe` logic in the Makefile.
And if someone just grabs a tar or zip file with the latest commit (i.e., you are outside of a git repository) this will not help, either.